src/middleware/version.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextFunction, Request, Response } from "express";
+import { logMessage } from "@lib/logging/logger.js";
+
+export const versionMiddleware = (version: number) => {
+  return (req: Request, _: Response, next: NextFunction) => {
+    logMessage.debug(
```

Comment:
Should not we use `ìnfo` instead of `debug` here?

```suggestion
    logMessage.info(
```

Reply:
In order to keep noise to a minimum in Production and Staging I was thinking it would be helpful perhaps when developing locally but not required when running in Staging / Production.